src/LightPanels/LightPanels.js

```diff
@@ -805,7 +809,8 @@ module.exports = kind(
 		this._currentPanel = nextPanel;
 
 		// ensure that `transitionFinished` is called in the case where we are not animating
-		if (!this.shouldAnimate() || !animate) this.transitionFinished(null, null, true);
+		if (!this.shouldAnimate() || !animate) this.transitionFinished();
```

Comment:
This is more a stylistic question but I've been wondering if we should get out of the habit of calling event handlers directly and instead refactor the logic into another private method. It's minor in this case but allows us to avoid conditional arg requests (`(!ev || ev.originator === this.$.client)`). We'd still need similar logic in the handler but the shared method would just guard on `this.transitioning`.

wdyt?